Jagadeesh_Assignment_27_35min

JagadeeshMJun20 opened the pull request:
Please review the code.
Output:
![image](https://user-images.githubusercontent.com/66856654/87445361-98e55280-c615-11ea-85bd-7ee25a516ba8.png)

MadhuraMJun20 commented on src/jagadeeshMJun20/MeetingRoom.java:
```diff
@@ -0,0 +1,98 @@
+			System.out.println("Nalanda meeting room is booked for " + nalandaBookedHours + " hours");
+		if (takshasilaAvail != true)
+			System.out.println("Takshasila meeting room is booked for " + takshasilaBookedHours + " hours");
+		if (agraFortAvail != true)
```
Here, I think that instead of all this logic a simple System.out.println() statement printing instance variable values would be fine.

JagadeeshMJun20 commented:
Updated the code and incorporated enum. Please review.
Output:
![image](https://user-images.githubusercontent.com/66856654/87552075-d35ef600-c6ce-11ea-95ac-0b6ae47a0dfe.png)

MadhuraMJun20 commented on src/jagadeeshMJun20/MeetingRoom.java:
```diff
@@ -0,0 +1,94 @@
+		System.out.println("====================================");
+		if (nalandaAvail != true)
+			System.out.println("Nalanda meeting room is booked for " + nalandaBookedHours + " hours");
+		if (takshasilaAvail != true)
```
here you can also write if (! takshashilaAvail).

MadhuraMJun20 commented on src/jagadeeshMJun20/MeetingRoom.java:
```diff
@@ -0,0 +1,94 @@
+	void bookMeetingRoom(MEETINGROOMS meetRoom, int hours) {
+		if (meetRoom == MEETINGROOMS.TAKSHASILA && takshasilaAvail == true) {
+			takshasilaAvail = false;
+			takshasilaBookedHours += hours;
```
I think here takshashilaBookedHours = hours; is sufficient for assigning value of hours into static var.

MadhuraMJun20 commented on src/jagadeeshMJun20/MeetingRoom.java:
```diff
@@ -0,0 +1,94 @@
+	static boolean nalandaAvail = true;
+	static boolean takshasilaAvail = true;
+	static boolean agraFortAvail = true;
+	static boolean pratapGadhAvail = true;
```
You can declare these variables in single line since they are of same type.
